website/views/shared/header/header.jade

```diff
@@ -12,23 +12,25 @@
       .meter.health(tooltip='{{Math.round(user.stats.hp * 100) / 100}}')
         .bar(ng-style='{"width": Shared.percent(user.stats.hp, Shared.maxHealth)+"%"}')
         span.meter-text.value
-          | {{Math.ceil(user.stats.hp)}} / {{::Shared.maxHealth}}
+          | {{statCalc.hpDisplay(user.stats.hp)}}
```

Comment:
the hpDisplay function is the only one out of the the statServices functions that takes in the specific current hp value rather than a `user` representation. I wanted to refactor this but it was outside the scope of this PR.